docs/en/Audit-Logging.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: global
+title: Audit Logging
+group: Features
+priority: 1
+---
+
+* Table of Contents
+{:toc}
+
+## Overview
+Alluxio supports audit logging of user access to your data. With audit logging, system administrators can
+keep track of each user's access to data in Alluxio.
+
+## Enable Audit Logging
+To enable Alluxio audit logging, you need to set the JVM property `alluxio.master.audit.logging.enabled` to true. Besides,
+you also need to configure the proper log4j appender type for the audit log. Alluxio already provides a default log
+appender called `MASTER_AUDIT_LOGGER`. This is sufficient for most users. Advanced users can use their own appender and set
+the environment variable `${ALLUXIO_MASTER_AUDIT_LOGGER}` to the name of their custom appender.
+In `./conf/alluxio-env.sh`, add the following two lines:
+
+```bash
+ALLUXIO_MASTER_JAVA_OPTS+=" -Dalluxio.master.audit.logging.enabled=true"
+ALLUXIO_MASTER_JAVA_OPTS+=" -Dalluxio.master.audit.logger.type=${ALLUXIO_MASTER_AUDIT_LOGGER:-MASTER_AUDIT_LOGGER}"
+```
+Then you need to restart Alluxio, and you will find a file named `master_audit.log` in Alluxio's logs directory.
+
+
+## Alluxio Audit Log Format
+The format of Alluxio audit log file (`master_audit.log`) is very similar to HDFS's audit log file, as shown in
```

Comment:
nit: add a link to HDFS audit log doc? since we mentioned it here, do we also want to point out the difference between Alluxio audit and HDFS audit logging?

Reply:
I found this [doc](https://wiki.apache.org/hadoop/HowToConfigure), but it seems it is outdated. Other than this, I haven't found the recent one.

Reply:
Done.
I used this link, because from their latest doc, I could find a way to navigate to that page.